test/sql/catalog/comment_on_extended.test

```diff
@@ -0,0 +1,103 @@
+# name: test/sql/catalog/comment_on_extended.test
+# description: Test COMMENT ON to add comment on database
+# group: [catalog]
+
+require skip_reload
+
+### Create some test data
+statement ok
+ATTACH '__TEST_DIR__/comment_on_extended_1.db' AS db1
+
+statement ok
+ATTACH '__TEST_DIR__/comment_on_extended_2.db' AS db2
+
+statement ok
+CREATE SCHEMA db1.s1;
+
+statement ok
+CREATE SCHEMA db2.s2;
+
+statement ok
+CREATE TABLE db1.s1.t1 AS SELECT 1 as c1
+
+statement ok
+CREATE TABLE db2.s2.t2 AS SELECT 2 as c2
+
+# some extra tables to index on (since we can not comment on tables with dependencies)
+statement ok
+CREATE TABLE db1.s1.t3 AS SELECT 3 as c3
+
+statement ok
+CREATE TABLE db2.s2.t4 AS SELECT 4 as c4
+
+statement ok
+CREATE INDEX test_index ON db1.s1.t3 using art(c3)
+
+statement ok
+CREATE INDEX test_index ON db2.s2.t4 using art(c4)
+
+### Confirm we have no comments yet
+query I
+SELECT count(*) from duckdb_tables() where comment IS NOT NULL;
+----
+0
+
+query I
+SELECT count(*) from duckdb_columns() where comment IS NOT NULL;
+----
+0
+
+### Tables with fully qualified names
+statement ok
+COMMENT ON TABLE db1.s1.t1 IS 'very gezellige table 1'
```

Comment:
Can we also test partially qualified names that are unambiguous, e.g. `COMMENT ON TABLE db1.t1` and `COMMENT ON TABLE s1.t1`?